src/Core/src/Core/ITitleBar.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Titlebar for the Window
+	/// </summary>
+	public interface ITitleBar : IView
+	{
+		IView? Content { get; }
```

Comment:
Review if these properties need to be broken out as separate properties?

Maybe this could be an array of views?
Array of rects?